source/draw/gpu/opengl/backend/gles3/Gles3Debug.ooc

```diff
@@ -57,7 +60,7 @@ getEglErrorMessage: func (errorCode: Int) -> String {
 		case 12300 => "EGL_BAD_PARAMETER"
 		case 12301 => "EGL_BAD_SURFACE"
 		case 12302 => "EGL_CONTEXT_LOST"
-		case => "Unknown ERROR CODE: " + errorCode toString()
+		case => "Unknown ERROR CODE: %s " format(errorCode toString())
```

Comment:
Not really, either way is fine. Using the `&` operator gives a slightly shorter line.